flutter/wtc/lib/app.dart

```diff
@@ -173,7 +178,8 @@ class _NavBars extends State<App> {
         const MapPage(),
         const HomePage(),
         const CalendarPage(),
-        const AccountPage()
+        const AccountPage(),
+        const CreatePostPage()
```

Comment:
can remove this since the hamburger menu button doesn't access the body to get to create post page.

Reply:
Removed